voluncheer/map/admin.py

```diff
@@ -1,3 +1,6 @@
-# from django.contrib import admin
+from django.contrib import admin
+
+from .models import NYCharities
```

Comment:
Please use absolute paths for imports:

```suggestion
from map.models import NYCharities
```